.golangci.yml

```diff
@@ -81,6 +81,13 @@ linters:
 #    - whitespace
     - wrapcheck
 
+issues:
+  exclude-rules:
+    # Disable funlen for "func Test..." or func (suite *Suite) Test..." type functions
+    # These functions tend to be descriptive and exceed length limits.
+    - source: "^func (\\(.*\\) )?Test"
+      linters:
+        - funlen
```

Comment:
rather than regex
```
issues:
  exclude-rules:
    - linters:
        - funlen
      path: _test\.go$
```
`path: _test\.go$ specifies that the exclusion should apply to files ending in _test.go.`

Reply:
I thought about it, but it'd still be nice to validate function length across all other functions, including those defined in test files.  We should hold test and non-test code to the same standard, with the exception being functions that just reference test cases.

In addition, the regex above is smallest exception we can make, and unless we have a strong reason to, imo it's best to relax linting rules in small increments.